blocks/right-rail-advanced-block/layouts/right-rail-advanced/default.jsx

```diff
@@ -31,7 +31,7 @@ const RightRailAdvancedLayout = ({ children }) => {
   return (
     <>
       <header className="page-header">{navigation}</header>
-      <section role="main" className="main">
+      <section role="main" id="main" className="main" tabIndex="-1">
```

Comment:
I used a simple `a` tag here since we have issues using any of our ES6 packages in OutputTypes: https://arcpublishing.atlassian.net/browse/PEN-1545

This is an ongoing issue, but as far as using PrimaryFont is concerned, product and design both agreed to default to `Arial` font for now and use the Large Outlined Button for general styling guidelines.